taiga/auth/services.py

```
@@ -104,107 +131,171 @@ def refresh_token(refresh_token: str):
+        verified_email=False,
+        read_new_terms=True
+    )
 
     user.set_password(password)
     try:
         user.save()
-    except IntegrityError:
-        raise exc.WrongArguments(_("Error while creating new user."))
+        logger.info("User registered via private invitation: %s (email: %s)", username, email)
+    except IntegrityError as e:
+        logger.error("Database integrity error during private user registration: username=%s, email=%s", username, email)
```

Ruff (0.14.4)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)